src/processor/Package.ts

```diff
@@ -83,4 +84,37 @@ export class Package {
       pullAt(sd.rules, rulesToRemove);
     });
   }
+
+  // If an extension only has a single context, and it is the default context, suppress it.
+  private removeDefaultExtensionContextRules(): void {
+    this.extensions.forEach(sd => {
+      const numContexts = sd.rules.filter(
+        r =>
+          r instanceof ExportableCaretValueRule &&
+          r.path === '' &&
+          /^context\[\d+]\.type$/.test(r.caretPath)
+      ).length;
+      if (numContexts === 1) {
+        // * ^context[0].type = #element
+        const typeRuleIdx = sd.rules.findIndex(
+          r =>
+            r instanceof ExportableCaretValueRule &&
+            r.path === '' &&
+            r.caretPath === 'context[0].type' &&
+            isEqual((r as ExportableCaretValueRule).value, new fshtypes.FshCode('element'))
```

Comment:
I think TypeScript can infer that `r` is an `ExportableCaretValueRule`. It seems to be doing that on `112`.

Reply:
Yes, indeed.  That's the result of some refactoring where I initially had that `isEqual` in a separate `if` condition (without the `instanceof`, so it was necessary).  Thanks for finding that, eagle eye @ngfreiter!